Unpin ajv dependency so Greenkeeper can move to 5.1.6 once ajv fixes its issue.

https://github.com/feathersjs/feathers-hooks-common/issues/211

On latest node version only:

$ npm install 
npm ERR! code ETARGET
npm ERR! notarget No matching version found for json-schema-traverse@^0.3.0
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget 
npm ERR! notarget It was specified as a dependency of 'ajv'
npm ERR! notarget

--- a/package.json
+++ b/package.json
@@ -53,7 +53,7 @@
   },
   "homepage": "https://github.com/feathersjs/feathers-hooks-common#readme",
   "dependencies": {
-    "ajv": "5.2.0",
+    "ajv": "^5.2.0",
     "debug": "^2.2.0",
     "feathers-errors": "^2.4.0",
     "feathers-hooks": "^2.0.1",

